static/js/click_handlers.js

```diff
@@ -829,12 +829,12 @@ exports.initialize = function () {
             const edit_area = $(this).parent().find(`${selector}`);
             $(selector).removeClass("stream-name-edit-box");
```

Comment:
Ugh the `selector` here is being pulled from the DOM.  This goes back to 14b471a29f1736d60e73177d3dff9b609b89c8b8 from 2016.  I know this is outside the scope of this PR, but this code makes me feel queasy.

~~~
+      <span class="stream-name-editable editable-section">{{name}}</span>
+      {{#if is_admin}}
+        <span class="editable" data-make-editable=".stream-name-editable"></span>
+        <span class="checkmark" data-finish-editing=".stream-name-editable">✓</span>
+      {{/if}}

Reply:
I share your queasiness.

Reply:
https://github.com/zulip/zulip/issues/17211

cc @aero31aero (more settings code to clean up)